### ISSUE TITLE
Window scaling issue in Windows 11

### ISSUE THREAD

**Issue by kmdono02** [shinyquagsire23/OpenJKDF2#279]
Hi everyone! Thanks for this amazing mod.  I'm having an issue with window scaling when running OpenJKDF2.  Specifically unless the window size is of a specific ratio, the picture appears incorrectly scaled.  This is also the case with fullscreen on my 2K monitor.  Posted a picture as an example.  I tried messing around with the DPI setting on OpenJKDF2 and the DPI compatibility settings in Windows 11 and running it in compatibility mode.  Any help is appreciated!

![Screenshot 2023-08-31 002349](https://github.com/shinyquagsire23/OpenJKDF2/assets/43554362/6c4a4a4c-f272-4db6-a6dd-233e758e7487)

**Comment by kmdono02** [comment]
example of size which works ok.  Have also tried changing Windows scaling %
![Screenshot 2023-08-31 002454](https://github.com/shinyquagsire23/OpenJKDF2/assets/43554362/87d9fd34-fd2d-4ad9-9cf1-51d64341688a)

**Comment by shinyquagsire23** [comment]
sorry for the late reply, what GPU do you have? It looks like it's possibly expecting powers of 2 for the internal resolution, so I'll have to see if I can fix it blindly based on that, but if possible I'd like to recreate the issue on any hardware I have access to

**Comment by kmdono02** [comment]
No problem thanks for getting back to me!  The GPU is an integrated graphics card - Radeon 780M.  It's really new which may be a part of the problem?

FYI I tried OpenJKDF2 on my old laptop with Windows 10 and an integrated Intel HD 620 GPU and it works perfectly, so I can always use my old laptop to play but of course the new computer is preferred

Also not sure if it relevant by I've tried setting my monitor to 1080p and 1440p and doing full screen, both result in the same issue above
